docs/start/getting-started/fragments/flutter/add-api.md

```diff
@@ -1,65 +1,143 @@
-### Setup AWS Cloud Resources with Amplify CLI 
+## Deploy your Amplify sandbox backend
 
-We will now use the Amplify CLI to configure the AWS Cloud Resources that will power your app. 
+<amplify-callout>
 
-1. First install the amplify cli. Within your terminal run: 
+If you chose to generate your models earlier via the CLI, you can skip this step and proceed to [Add authentication](~/start/getting-started/add-api.md#add-authentication).
```

Comment:
Following my comment about the multiple options, keeping it opinionated towards Admin UI would avoid another branch out at this point. @renebrandel